dashboard/model/influxdb_data.py

```diff
@@ -11,8 +11,9 @@ def get_influx_stats(self, table, days):
         return list(self.influx.query(query).get_points())
 
     def __get_table_sql(self, table, days):
-        query = f'SELECT records FROM "emr_stats_{table.db}_{table.name}" ' \
-                f'WHERE time >= now() - {days}d ' \
+        period_filter = f' AND period_id = \'{table.period.id}\' ' if table.period else ''
```

Comment:
Can we change it to triple quotes and avoid excape signs inside?

Reply:
Good idea, even standard quotes will work here, thanks!